sideboard/websockets.py

```diff
@@ -102,8 +114,22 @@ def _normalize_channels(*channels):
 
 
 def notify(channels, trigger="manual", delay=0, originating_client=None):
-    broadcaster.delayed(delay, _normalize_channels(*sideboard.lib.listify(channels)),
-                        trigger=trigger, originating_client=originating_client or threadlocal.get_client())
+    """
+    Manually trigger all subscriptions on the given channels.  The following
+    optional parameters may be specified:
+
+    trigger: Used in log messages if you want to distinguish between triggers.
+    delay: If provided, wait this many seconds before triggering the broadcast.
+    originating_client: Websocket subscriptions will NOT fire if they have the
+                        same client as the trigger.
+    """
+    channels = _normalize_channels(*sideboard.lib.listify(channels))
+    context = {
+        'trigger': trigger,
+        'originating_client': originating_client or threadlocal.get_client()
+    }
+    broadcaster.delayed(delay, channels, **context)
+    local_broadcaster.delayed(delay, channels, **context)
```

Comment:
One new websocket-related feature we added is the ability to say "call this local function anytime a particular websocket event is fired".